### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

File: model/SharedLibrariesRegistry.php [hunk 1]
```diff
@@ -168,6 +168,8 @@ public function registerFromItem($path)
         $dom->load($path);
         $basePath = pathinfo($dom->documentURI, PATHINFO_DIRNAME);
         
+        $basePath = ltrim($basePath, 'file:/');
```
Comment (hutnikau) [inline model/SharedLibrariesRegistry.php]: `$dom->documentURI` returns path with protocol (`file:/`).

File: model/qti/ParserFactory.php [hunk 2]
```diff
@@ -1483,7 +1486,7 @@ private function buildInfoControl(DOMElement $data){
 
             $ciClass = '';
             $classes = $data->getAttribute('class');
-            $classeNames = split('/\s+/', $classes);
+            $classeNames = preg_split('/\s+/', $classes);
```
Comment (hutnikau) [inline model/qti/ParserFactory.php]: `split()` function is deprecated.

File: model/qti/templates/interactions/qti.customInteraction.tpl.php [hunk 3]
```diff
@@ -24,9 +24,9 @@
         <pci:responseSchema href="http://imsglobal.org/schema/json/v1.0/response.json"/>
         <pci:resources location="http://imsglobal.org/pci/1.0.15/sharedLibraries.xml">
             <pci:libraries>
-                <?foreach(get_data('libraries') as $lib):?>
+                <?php foreach(get_data('libraries') as $lib):?>
                 <pci:lib id="<?=$lib?>"/>
-                <?endforeach;?>
+                <?php endforeach;?>
             </pci:libraries>
```
Comment (hutnikau) [inline model/qti/templates/interactions/qti.customInteraction.tpl.php]: Short tags are deprecated.

File: model/qti/templates/qti.item.tpl.php [hunk 4]
```diff
@@ -23,7 +23,9 @@
 <assessmentItem 
     xmlns="http://www.imsglobal.org/xsd/imsqti_v2p1"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    <?php foreach(get_data('namespaces') as $name => $uri):?>xmlns:<?=$name?>="<?=$uri?>"<?php endforeach?>
+    <?php foreach(get_data('namespaces') as $name => $uri):?>
```
Comment (hutnikau) [inline model/qti/templates/qti.item.tpl.php]: This loop generated invalid namespace attributes (without spaces and line breaks):
`xmlns:a="..."xmlns:b=".."`

**Comment by ssipasseuth** [comment]
I have an error running the unit test LocalSharedLibrariesTest::testRegisterFromItem (do you have it on your local setup too ?)

1) LocalSharedLibrariesTest::testRegisterFromItem with data set #1 ('/home/sam/projects/TAO/packag...em.xml', array('OAT/shapes/collisions', 'OAT/shapes/shapes', 'OAT/shapes/shapes'))
oat\taoQtiItem\model\SharedLibraryNotFoundException: Shared Library could not be found at location 'home/sam/projects/TAO/package-tao/taoQtiItem/test/samples/sharedLibraries/registry_with_unofficial/OAT/shapes/collisions.js'.

/home/sam/projects/TAO/package-tao/taoQtiItem/model/SharedLibrariesRegistry.php:124
/home/sam/projects/TAO/package-tao/taoQtiItem/model/SharedLibrariesRegistry.php:187
/home/sam/projects/TAO/package-tao/taoQtiItem/test/SharedLibrariesRegistryTest.php:121

**Comment by hutnikau** [comment]
I have replaced `ltrim()` to `preg_replace()` to remove `file:/` prefix from file path. Now unit test should work fine.

**Pull request merged**